src/routes/Projects/routes/Project/components/Editor/Editor.js

```diff
@@ -141,7 +141,10 @@ export default class Editor extends Component {
 
   render () {
     return (
-      <div className={classes['container']} id={this.props.name} />
+      <section className={classes['editor']}>
+        <style>{'.CodeMirror,.CodeMirror-scroll,.firepad {overflow: hidden; overflow-y: scroll; position: relative; height: 100%; width: 100%;}'}</style>
```

Comment:
We should probably place this in root level `scss`. Lets create `styles/_editor-global.scss` or similar and include this in there. Then in `core.scss` we can import it (so it is available globally) or we can use `import 'styles/_editor-global.scss'` within editor, so that the global css is only applied when `Editor` is loaded.